Modifica modulo

eshiol opened the pull request:
### Summary of Changes
Aggiunto layout pagina modifica modulo

### Testing Instructions
Accedere al frontend con un utenza di amministrazione ed editare un modulo qualsiasi
![screencapture-design-italia-marcogargano-2019-02-06-22_59_35](https://user-images.githubusercontent.com/12718836/74465363-8b287700-4e95-11ea-99e9-e33bc71e180c.png)

### Expected result
![screencapture-localhost-pasw-italiapasw14-1-index-php-2020-02-13-19_10_30](https://user-images.githubusercontent.com/12718836/74465413-a1cece00-4e95-11ea-9dd5-5ea6b1bc0a9e.png)

### Actual result
![screencapture-localhost-pasw-italiapasw14-1-index-php-2020-02-13-19_07_56](https://user-images.githubusercontent.com/12718836/74465423-a6938200-4e95-11ea-8b08-2d3fab30fe22.png)

### Documentation Changes Required

eshiol committed:
modifica modulo

NioBee commented:
Applicata Commit SHA: cdd9742ce1

![image](https://user-images.githubusercontent.com/41695549/74468509-3b4cae80-4e9b-11ea-9c1c-167a7e87e7bf.png)

Nelle opzioni di modifica mi pare ci sia ancora qualche piccola sbavatura:
![image](https://user-images.githubusercontent.com/41695549/74468713-91b9ed00-4e9b-11ea-9b2a-bc42b02a4bce.png)

Queste 2 voci non hanno la descrizione nè un tooltip quando si passa con il mouse:
(forse perchè sono non editabili, però allora sarebbe carino ci fosse scritto da qualche parte..)
![image](https://user-images.githubusercontent.com/41695549/74468797-bca44100-4e9b-11ea-9618-d0026b5af9bf.png)

eshiol commented:
Confermo la mancanza formattazione del campo modal_menu

dspina72 commented:
![2020-02-14_1017](https://user-images.githubusercontent.com/15988690/74522509-9d4cf880-4f1b-11ea-8a87-ccfa812d42ae.png)
![2020-02-14_1010](https://user-images.githubusercontent.com/15988690/74522529-a3db7000-4f1b-11ea-9c57-c50445d8602c.png)

mancanza di formattazione in lead e footermenu

eshiol committed:
fix modal_menu field

NioBee commented:
Applicata Commit SHA: a2fb1160fd

![image](https://user-images.githubusercontent.com/41695549/74605946-a3cea200-50cc-11ea-858d-844eb11cdb21.png)

Patch successful!

PS nel modulo newsflash il carattere usato in questo box sembra diverso dagli altri...
![image](https://user-images.githubusercontent.com/41695549/74605980-0031c180-50cd-11ea-9a9e-5c396acebd1b.png)

nel modulo carousel questa parte mi pare un po' strana...
![image](https://user-images.githubusercontent.com/41695549/74606038-661e4900-50cd-11ea-8463-cc0459d0f88c.png)
Forse sarebbe utile aggiungere un tooltip a quelle iconcine.. non sono subito chiare..

eshiol committed:
subform repeatable

NioBee commented:
Applicata Commit SHA: 4a8af0ef17

![screenshot](https://user-images.githubusercontent.com/41695549/74874065-97fb0e00-5360-11ea-90a0-ecd22e89c2c3.png)

Risolta la visualizzazione del modulo carousel
Patch successful

NioBee commented:
Trovo che nel modulo carousel sarebbe utile aggiungere un piccolo tooltip nei tre pulsanti di gestione slides.. ad esempio potrebbero essere i seguenti:

+ "aggiungi - aggiungi una slide alla galleria" (add - add a slide to the gallery)
x "elimina - elimina questa slide dalla galleria" (delete - delete this slide from the gallery)
-> "riposiziona - trascina per riposizionare questa slide" (rearrange - drag to move this slide)

eshiol commented:
I pulsanti non sono legati al modulo carousel. Vengono generati per tutti i campi presenti in una sub-form. Direi di usare solo la label: Aggiungi; Rimuovi; Sposta.

eshiol committed:
allineamento pulsanti

eshiol committed:
tooltips

eshiol committed:
repeatable-table

NioBee commented:
> I pulsanti non sono legati al modulo carousel. Vengono generati per tutti i campi presenti in una sub-form. Direi di usare solo la label: Aggiungi; Rimuovi; Sposta.

perfetto, non lo sapevo.. credevo fossero specifici.. 😅

eshiol committed:
disabled joomla.form.field.subform.repeatable-table groupByFieldset=true

sostituito temporanemente il layout="joomla.form.field.subform.repeatable-table" groupByFieldset="true" con layout="joomla.form.field.subform.repeatable-table" groupByFieldset="false"

dspina72 commented:
formattazione in lead e footermenu Applicata Commit SHA: 3a7355c Patch successful
